learning-bitmark/quick-start/simple-solution-for-node-setup.md

```diff
@@ -1 +1,85 @@
-#  [Learning Bitmark / Quick start] - Simple solution for node setup
+#  [ Learning Bitmark / Quick start ] - Simple solution for node setup
+
+## Start your bitmark node
+
+A user can run a bitmark node in several ways. This section will guide you to start a bitmark node using docker container. Inside a bitmark node docker container, there are bitmarkd, recorderd, payment system and a user interface to control and monitor services.
+
+## Prerequisites
+
++ [Docker](https://docs.docker.com/install/) is installed in your system
+    +  Windows user run docker on Hyper-V
```

Comment:
This seems confusing: readers won't necessarily understand it, and it's already covered in the Docker docs (in fact, it looks like the Docker install automatically turns on Hyper-V)

Reply:
@shannona  some widows do not support hyper-v ie, home-edition but I can not figure out them all. It depends on users to find out themselves. What is your suggestion for this situation?